microcosm_postgres/types/__init__.py

```diff
@@ -0,0 +1,5 @@
+from microcosm_postgres.types.enum import EnumType  # noqa: F401
+from microcosm_postgres.types.json import JSONType  # noqa: F401
+from microcosm_postgres.types.serial import Serial  # noqa: F401
+from microcosm_postgres.types.utc_date_time import EPOCH, UTCDateTime, utcnow  # noqa: F401
+from microcosm_postgres.types.uuid import UUIDType  # noqa: F401
```

Comment:
Whenever SQLAlchemy-utils will fix their support for SQLAlchemy, we can import `JSONType` and `UUIDType` from SQLAlchemy-utils here.

Reply:
IMO we use `sqlalchemy-utils` so little in our codebase, I would be okay with dropping the dependency altogether.